spec.md

```diff
@@ -0,0 +1,100 @@
+# bimg File Format Specification
+bimg (abbreviation of "blit image") is a versatile image format for the ComputerCraft Minecraft mod.
+This specification aims to provide a standardized image format that can be easily implemented and used throughout the entire ComputerCraft ecosystem.
+
+# File Structure
+The examples shown throughout this specification are based on a sample image file, serialized using `textutils.serialize`.
+
+An image file can be separated into two sections: metadata and frame data:
```

Comment:
Might be a good idea to explicitly state that metadata and frame data are unordered, just in case someone tries to parse it without using Lua.